Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -45,6 +45,37 @@
 
 THREAT_INDICATORS_HEADERS = ['Name', 'DisplayName', 'Values', 'Types', 'Source', 'Confidence', 'Tags']
 
+# =========== Mirroring Mechanism Globals ===========
+
+MIRROR_DIRECTION_DICT = {
+    'None': None,
+    'Incoming': 'In',
+    'Outgoing': 'Out',
+    'Incoming And Outgoing': 'Both'
+}
+
+MIRROR_STATUS_DICT = {
+    'Undetermined': 'Other',
+    'TruePositive': 'Resolved',
+    'BenignPositive': 'Resolved',
+    'FalsePositive': 'False Positive',
+}
+
+MIRROR_DIRECTION = MIRROR_DIRECTION_DICT.get(demisto.params().get('mirror_direction'))
+INTEGRATION_INSTANCE = demisto.integrationInstance()
+
+INCOMING_MIRRORED_FIELDS = ['ID', 'Etag', 'Title', 'Description', 'Severity', 'Status', 'Label', 'FirstActivityTimeUTC',
+                                  'LastActivityTimeUTC', 'LastModifiedTimeUTC', 'CreatedTimeUTC', 'IncidentNumber', 'AlertsCount',
+                                  'AlertProductNames', 'Tactics', 'relatedAnalyticRuleIds', 'IncidentUrl', 'classification',
+                                  'classificationComment', 'alerts', 'entities', 'comments', 'relations']
+OUTGOING_MIRRORED_FIELDS = {'etag', 'title', 'description', 'severity', 'status', 'labels', 'firstActivityTimeUtc',
+                            'lastActivityTimeUtc', 'classification', 'classificationComment', 'classificationReason'}
```

Comment:
why INCOMING_MIRRORED_FIELDS is list and OUTGOING_MIRRORED_FIELDS is set?

Reply:
Decision of the server, even though it says [here](https://xsoar.pan.dev/docs/integrations/mirroring_integration#get-mapping-fields) that it can be a list